CMakeLists.txt

```diff
@@ -163,9 +163,12 @@ endif()
 
 set_target_properties(${STLINK_LIB_STATIC} PROPERTIES OUTPUT_NAME ${PROJECT_NAME})
 
-install(TARGETS ${STLINK_LIB_STATIC}
-	ARCHIVE DESTINATION ${STLINK_LIBRARY_PATH}
-)
+set(STATIC_LIB_INSTALL ON CACHE BOOL "Install static lib")
+if (STATIC_LIB_INSTALL)
+    install(TARGETS ${STLINK_LIB_STATIC}
```

Comment:
We should not build the static library at all when `STLINK_STATIC_LIB` is not set to `ON`. We should also document this behaviour and set it default to OFF. See as an example https://github.com/nanomsg/nanomsg/blob/309af15f808546f5f04b0761d7685b90e6bcbb94/README.md#static-library

Reply:
I am try disable build static lib but fail because it used somwhere in tools. And I not understand how to disable it clearly. Thats why I am only disable install of them.

Reply:
You are right, currently it is a bit of a mess because for windows and mac builds we use the static build (easy distribution) and for others we use the dynamic library build. We should keep it this way. Then you can ignore my comment above.

Reply:
OK